assets/js/components/ExecutionResults/ExecutionResults.jsx

```diff
@@ -3,17 +3,21 @@ import Table from '@components/Table';
 
 import ReactMarkdown from 'react-markdown';
 import remarkGfm from 'remark-gfm';
-import { EOS_ERROR } from 'eos-icons-react';
 import LoadingBox from '@components/LoadingBox';
-import NotificationBox from '@components/NotificationBox';
 
+import {
+  REQUESTED_EXECUTION_STATE,
+  RUNNING_EXECUTION_STATE,
+} from '@state/lastExecutions';
 import { getCheckResults, getCheckDescription } from './checksUtils';
 
 import ResultsContainer from './ResultsContainer';
 import { ExecutionIcon } from './ExecutionIcon';
 import CheckResultOutline from './CheckResultOutline';
 import ExecutionHeader from './ExecutionHeader';
 
+const runningStates = [RUNNING_EXECUTION_STATE, REQUESTED_EXECUTION_STATE];
```

Comment:
It looks a bit strange that we need to import the same constants we do in the `ExecutionResultsPage` here.
Maybe it would make more sense to add other extra paremeter to `ExecutionResults`, like `executionRunning`, or simply send the `executionData.status` and do the logic only in one file, not in both.
Because I see the `executionStarted={executionData?.status !== REQUESTED_EXECUTION_STATE}` in the other and `  if (runningStates.includes(executionData?.status)) {` here, and both have the same intention

Reply:
Yes, makes sense. Addressing.